addons/mail/static/src/components/composer/composer.scss

```diff
@@ -13,258 +14,76 @@
-    }
+.o_Composer_textInput:not(.o-composer-is-compact) {
+    min-height: $o-mail-composer-text-input-height;
 }
 
-.o_Composer_threadTextualTypingStatus {
-    font-size: $font-size-sm;
-
-    &:before {
-        // invisible character so that typing status bar has constant height, regardless of text content.
-        content: "\200b"; /* unicode zero width space character */
-    }
+.o_Composer_threadTextualTypingStatus:before {
+    // invisible character so that typing status bar has constant height, regardless of text content.
+    content: "\200b"; /* unicode zero width space character */
 }
 
-.o_Composer_toolButton {
-    // keep a margin between the buttons to prevent their focus shadow from overlapping
-    margin-left: map-get($spacers, 1);
-    margin-right: map-get($spacers, 1);
-}
-
-.o_Composer_toolButtons {
-    display: flex;
-    padding-top: map-get($spacers, 1);
-    padding-bottom: map-get($spacers, 1);
-
-    &:not(.o-composer-is-compact) {
-        flex-direction: row;
-        justify-content: space-between;
-        flex: 100%;
-    }
-}
-
-.o_Composer_toolButtonSeparator {
-    flex: 0 0 auto;
-    margin-top: map-get($spacers, 2);
-    margin-bottom: map-get($spacers, 2);
+.o_Composer_toolButtons:not(.o-composer-is-compact) {
+    flex-basis: 100%;
 }
 
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
-// TODO FIXME o-open on the button should be enough.
-// Style of button when popover is "open" comes from web.Popover, and we can't
-// define a modifier on .o_Composer_button due to not being aware of Popover's
-// state in context of template. https://github.com/odoo/owl/issues/693
-.o_is_open .o_Composer_toolButton {
-    background-color: gray('200');
-}
-
-.o_Composer {
-    &:not(.o-messaging-in-editing) {
-        background-color: gray('100');
-    }
-
-    &.o-is-in-thread-view {
-        background-color: $white;
-    }
-}
-
-.o_Composer_actionButton.o-last.o-has-current-partner-avatar.o-composer-is-compact {
-    border-radius: 0 $o-mail-rounded-rectangle-border-radius-lg $o-mail-rounded-rectangle-border-radius-lg 0;
-}
-
-.o_Composer_button.o-composer-is-compact {
-    border-left: none; // overrides bootstrap button style
-
-    :last-child {
-        border-radius: 0 3px 3px 0;
-    }
-}
-
 .o_Composer_buttonDiscard {
-    border: $border-width solid lighten(gray('400'), 5%);
-}
-
-.o_Composer_buttons {
-    border: 0;
-
-    &.o-messaging-in-editing {
-        border-right: $border-width solid $border-color;
-    }
-}
-
-.o_Composer_coreMain:not(.o-composer-is-compact) {
-    background: $white;
-    border: $border-width solid $border-color;
-    border-radius: $o-mail-rounded-rectangle-border-radius-lg;
+    border: $border-width solid lighten($gray-400, 5%);
 }
 
 .o_Composer_currentPartnerAvatar {
     object-fit: cover;
 }
 
-.o_Composer_textInput {
-    appearance: none;
-    outline: none;
```

Comment:
I removed `appearance` and `outline` because `o_Composer_textInput` is applied on a `div`, which already has these css values by default.